The Ember Times No. 205 - January 29th 2023 (#1229)
ember-learn/ember-blog: content/the-ember-times-issue-205.md

---
title: The Ember Times - Issue No. 205
authors:
  - jared-galanis
  - bryan-mishkin
  - anne-greeth-schot-van-herwijnen
  - chris-ng
date: 2022-11-20T00:00:00.000Z
tags:
  - newsletter
  - '2022'
---

👋 Emberistas! 🐹

> ember-template-lint 5.0 released 🧹, EmberConf News 🎤, new addons ✨, Ember 4.0+ upgrade survey 🐹, call for support with TS suppor in Ember 📦, updated addons 📬, Wacky Tricks We Use in Publishing TypeScript Types ▶️,

Reviewer: ```suggestion
ember-template-lint 5.0 released 🧹, EmberConf News 🎤, new addons ✨, Ember 4.0+ upgrade survey 🐹, call for support with TS support in Ember 📦, updated addons 📬, Wacky Tricks We Use in Publishing TypeScript Types ▶️,
```

<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

---

## [ember-template-lint 5.0.0 released 🧹](https://github.com/ember-template-lint/ember-template-lint/releases/tag/v5.0.0)

[Version 5.0](https://github.com/ember-template-lint/ember-template-lint/releases/tag/v5.0.0) has been released for [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint), the linter for best practices with Ember handlebars templates.

The key changes include:

- Linting embedded templates by default
- Adding nearly a dozen rules to the `recommended` config
- Changing a few rule names and option defaults
- Dropping support for old versions of Node

---

## 🎤 EmberConf News

> We know folks are getting excited about EmberConf, and might be wondering when it's happening. Currently, we're still getting our ducks in a row. After three years of virtual EmberConf, we're working out a way to match everybody's expectations around a possible hybrid conference. In the mean time, you can definitely keep working on those proposals and we’ll release more details as soon as we can. Historically EmberConf has happened in March, but this time it will be later in the year. Hope to see you there!

Reviewer 2: Noticed a few more grammar issues:

```suggestion
We know folks are getting excited about EmberConf and might be wondering when it's happening. Currently, we're still getting our ducks in a row. After three years of virtual EmberConf, we're working out a way to match everybody's expectations around a possible hybrid conference. In the meantime, you can definitely keep working on those proposals and we’ll release more details as soon as we can. Historically, EmberConf has happened in March, but this time it will be later in the year. Hope to see you there!
```

---

> ## ✨ The new addons in the block

Reviewer: ```suggestion
## ✨ The new addons on the block
```

I think?

## 🎤 EmberConf News

> We know everyone is getting excited about EmberConf and you might be wondering when it's coming. We are working on getting our ducks in a row. After three years of virtual EmberConf we're working out a way to match everybodies expectations around a possible hybrid conference. In the mean time you can definitely keep working on those proposals and we’ll release more details as soon as we can. Historically EmberConf has happend in March, but this time it will be later in the year. Hope to see you there!

Reviewer 2: Some grammar suggestions:

```suggestion
We know folks are getting excited about EmberConf, and might be wondering when it's happening. Currently, we're still getting our ducks in a row. After three years of virtual EmberConf, we're working out a way to match everybody's expectations around a possible hybrid conference. In the mean time, you can definitely keep working on those proposals and we’ll release more details as soon as we can. Historically EmberConf has happened in March, but this time it will be later in the year. Hope to see you there!
```

---

## 📬 Addon updates

* [ember-model-validator v4.4.0](https://github.com/esbanarango/ember-model-validator/releases/tag/v4.4.0) is now fully compatible with EmberJS 4.0+, the latest ember-data and adds Typescript support.

> * [ember-container-query v3.2.0](https://github.com/ijlee2/ember-container-query/releases) now supports Glint users. The PR commits ([#140](https://github.com/ijlee2/ember-container-query/pull/140/commits) & [#141](https://github.com/ijlee2/ember-container-query/pull/1401/commits)) serve as a learning material along the [offical documentation](https://typed-ember.gitbook.io/glint/).

Reviewer 3: Thank you for including my addon. :)

```suggestion
* [ember-container-query v3.2.0](https://github.com/ijlee2/ember-container-query/releases) now supports Glint users. The PR commits ([#140](https://github.com/ijlee2/ember-container-query/pull/140/commits) & [#141](https://github.com/ijlee2/ember-container-query/pull/141/commits)) serve as a learning material along the [offical documentation](https://typed-ember.gitbook.io/glint/).
```
